app/views/users/show.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2= "#{@user.name}'s video collections (#{@user.total_queue_items})"
+          = button_to "Follow", relationships_path(following_id: @user), class: "btn btn-default", style: "float: right;" unless @user == current_user || current_user.following?(@user)
+
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queued_item|
+              %tr
+                %td
+                  = link_to "#{queued_item.video.title}", queued_item.video
```

Comment:
Instead of this: `= link_to "#{queued_item.video.title}", queued_item.video` 

You can do `= link_to queued_item.video.title, queued_item.video`